assets/css/elements/_layout.scss

```diff
@@ -1,16 +1,22 @@
 html {
   box-sizing: border-box;
+  height: 100%;
+  width: 100%;
```

Comment:
I know this was existing and you just moved it up here, but I don’t _think_ `height` and `width` are needed and can be ✂️